assets/js/components/Dashboard/PaMessagesPage.tsx

```diff
@@ -141,7 +159,7 @@ const PaMessagesPage: ComponentType = () => {
                   className={cx("button", { active: stateFilter === "active" })}
                   onClick={() => setStateFilter("active")}
                 >
-                  Active
+                  Now
```

Comment:
Something I'm thinking about while working on the `End Now` task. We should also consider changing all references of `active` instead of just the label. This is a little more involved than normal because these values are passed to the server to help form the queries needed to filter the data. I still think it would be valuable to eliminate the inconsistent naming.

Reply:
I thought about that too, but `now` just seems a little awkward to me as an adjective. eg `get_now()` is a little weird compared to `get_active`. At the same time, it would be helpful to have two distinct words for describing messages that are within their active period and unpaused versus just generally within their active period. I sort of wish we used `current` rather than `now` or something.

Reply:
That's a good point. Maybe this is something to bring up to Mary? I didn't think about how it's kind of the outlier here not being an adjective (or at least an awkward one like you said). I agree with her that we can't use Active anymore, but your suggestion of Current sounds better to me.

Reply:
No reason we couldn't use `current` as our internal name and "Now" as the user-facing name.

Reply:
@digitalcora that works for me. I just pushed a change to use current internally.